dev/integration_tests/gradle_deprecated_settings/pubspec.yaml

```diff
@@ -9,15 +9,15 @@ dependencies:
     sdk: flutter
   camera: 0.10.5+9
 
-  camera_android: 0.10.8+17 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
+  camera_android: 0.10.8+18 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
```

Comment:
This is breaking tool integration tests

Reply:
Somehow this is still updated after I pinned this package in `update_packages_pins.dart`🧐